examples/apps/cchmc_ped_abd_ct_seg_app/dicom_text_sr_writer_operator.py

```
@@ -0,0 +1,586 @@
+        result_text = op_input.receive(self.input_name_dict)
+        if not result_text:
+            raise IOError("Input is read but blank.")
+
+        # Prepare content sequences elements (delegated to helper)
+        content_sequence_elements = self._create_content_sequence(result_text)
+
+        study_selected_series_list = None
+        try:
+            study_selected_series_list = op_input.receive(self.input_name_dcm_series)
+        except Exception:
+            pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)